java/example_code/cloudmap/src/main/java/aws/example/cloudmap/DiscoverInstanceRequest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Author : Rajib Deb
+ */
```

Comment:
Can you please add metadata to this code sample with the following license and description of code sample: 

```
// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
//
// This file is licensed under the Apache License, Version 2.0 (the "License").
// You may not use this file except in compliance with the License. A copy of
// the License is located at
//
// http://aws.amazon.com/apache2.0/
//
// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
// CONDITIONS OF ANY KIND, either express or implied. See the License for the
// specific language governing permissions and limitations under the License.

// snippet-sourcedescription:[DiscoverInstanceRequest.java demonstrates how to ....]
// snippet-service:[cloudmap]
// snippet-keyword:[java]
// snippet-keyword:[AWS Cloud Map]
// snippet-keyword:[Code Sample]
// snippet-sourcetype:[full-example]
// snippet-sourcedate:[2019-04-18]
// snippet-sourceauthor:[rajib76]
// snippet-start:[cloudmap.java.discover_instance_request.complete]
```

Reply:
done the changes